### DISCUSSION TITLE
Can tauri call android JAVA sdk?

### DISCUSSION THREAD

**Discussion by DennisZhangOiler** [tauri-apps/tauri#9418] [Q&A]
How can tauri 2 call android sdk? when will tauri 2 be stable?

**Comment by ellcs** [comment]
Hi @DennisZhangOiler,

To me it was also hard to find all the resources. I think this collection of links should help you:

  - https://beta.tauri.app/start/create-project/
  - https://beta.tauri.app/start/prerequisites/#configure-for-mobile-targets
  - https://v2.tauri.app/references/cli/#android

If you don't want to install android studio (as i tried to avoid it), you can build an apk with the following command:

> cargo tauri android build --target aarch64

The result apk might not be signed. You can sign it own you own with the apksigner from the android SDK. Maybe you can use the `~/.android/debug.keystore` to sign the apk. If it does not exist, generate one with the `keytool`.

  - https://developer.android.com/tools/apksigner?hl=de

I saved the links on the waybackmachine, because tauri's links seam to break from time to time.
